backend/lib/app.js

```diff
@@ -2,6 +2,27 @@
  * @param {import('fastify').FastifyInstance} fastify
  */
 export default async function (fastify) {
+  fastify.register(import('fastify-swagger'), {
+    routePrefix: '/docs',
+    openapi: {
+      info: {
+        title: 'CSAF Validator Service',
+        version: '1.0.0',
+        description:
+          'This is a service to validate documents against the CSAF standard.',
+        contact: {
+          url: 'https://github.com/secvisogram/csaf-validator-service',
+          email: 'secvisogram@bsi.bund.de',
+        },
+        license: {
+          name: 'MIT',
+          url: 'https://spdx.org/licenses/MIT.html',
```

Comment:
Is that correct that we link to the SPDX site or should we use https://github.com/secvisogram/csaf-validator-service/blob/main/LICENSE ? (Just a question - no preference or idea how it is used correctly.)

Reply:
Yes, probably it's better to refer to the project's license file.